src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
I will comment out the assert, since it makes debugging pretty hard.

I will also add a note to #8360 to uncomment the assert back once the bug is fixed so that it would guard against regressions.